Behat/Context/DrupalExtendedContext.php

```diff
@@ -497,4 +497,32 @@ public function userWithMailNotExists($mail) {
      $this->userWithMailExists($mail, FALSE);
    }
 
+  /**
+   * Checks that text appears before another text.
+   *
+   * Example:
+   * I should see "Element 1" text before "Element 2" from "li" element.
```

Comment:
As agreed by chat let's use a region instead of and HTML element to limit the scope.

Reply:
Example region: https://github.com/jhedstrom/drupalextension/blob/c4cee779960576830628fddb87cfebc7649166a9/src/Drupal/DrupalExtension/Context/MinkContext.php#L28